src/lib/utils/performance_warning.hpp

```diff
@@ -29,7 +29,8 @@ class PerformanceWarningClass {
  public:
   explicit PerformanceWarningClass(const std::string& text) {
     if (_disabled) return;
-    std::cout << "[PERF] " << text << "\n\tPerformance can be affected. This warning is only shown once." << std::endl;
+    std::cout << "[PERF] " << text << "\n\tPerformance can be affected. This warning is only shown once.\n"
```

Comment:
For some reason I never get a newline after a performance warning, even though there is `std::endl` here. This fixes that and shouldn't harm anyone.

Reply:
But now I will get two line breaks after a performance warning, right?

Reply:
I don't know, do you?

Reply:
because I don't, strangely